### PR TITLE
protip: export all from aws

### PR DESCRIPTION
**Pull request by atomantic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,15 @@ Commands:
   terraforming vpc             # VPC
 ```
 
+### Export all (example)
```
Comment (dtan4) [inline README.md]: IMO `example: Export all` is better.

File: README.md [hunk 2]
```diff
@@ -104,6 +104,15 @@ Commands:
...
+example assuming you want to export everything from us-west-2 and you are using ~/.aws/credentials with a `default` profile
```
Comment (dtan4) [inline README.md]: Please start sentence with capital character.

File: README.md [hunk 3]
```diff
@@ -104,6 +104,15 @@ Commands:
...
+```
```
Comment (dtan4) [inline README.md]: Add `bash` to enable syntax highlighting.

**Comment by dtan4** [comment]
This protip is just a example usage, so it is better to place `Export tfstate` section, I think.

Sorry for late review 🙇

**Comment by dtan4** [comment]
Please add TOC section at the top of README too.

**Comment by conatus** [comment]
Thanks for this. Was just about to write a shell command to do this, then saw you had already done it!

However, would it be plausible to add in the writing of `terraform.tfstate`? The problem I guess is you couldn't merge from an empty file, so maybe you could create a blank JSON file before your one liner...will have an experiment.

**Comment by atomantic** [comment]
@conatus 

You might find my newer method more useful: https://gist.github.com/atomantic/7d78ee36331c35d808476fec55c03386 -- my newer solution exports your whole region to a folder:

```
.
├── terraformingtool.sh
├── us-east-1
│   ├── global.tf
│   ├── main.tf
│   └── terraform.tfstate
└── us-west-2
    ├── global.tf
    ├── main.tf
    └── terraform.tfstate
```

global are aws components that are not specific to region, so if you need to just export one, you can still get your whole aws account saved.

**Comment by dtan4** [comment]
LGTM 👍  Thanks ❗ 
Sorry for late review 🙇

**Comment by conatus** [comment]
@atomantic Thanks for that - this looks super useful.

**Pull request merged**